web-ui/src/components/admin/users/Users.jsx

```diff
@@ -185,12 +185,8 @@ const Users = () => {
                           type: UPDATE_MEMBER_PROFILES,
                           payload: [...memberProfiles, data]
                         });
-                        if (member.pdlId) {
-                          emailPDLAssignment(member, csrf).then()
-                        }
-                        if (member.supervisorid) {
-                          emailSupervisorAssignment(member, csrf).then()
-                        }
+                        member.pdlId && emailPDLAssignment(member, csrf).then()
```

Comment:
Is the `.then()` necessary on these two lines?
It seems like what you're saying is that you don't want to wait for the result and when it finishes, you don't care if it succeeds or fails.
Perhaps these lines should be wrapped in a `try`/ `catch`, called with `await`, and output an error in `catch` if one of them fails.

Reply:
Added.